snapcraft/internal/errors.py

```diff
@@ -18,7 +18,15 @@
 from snapcraft import formatting_utils
 from snapcraft.internal import steps
 from subprocess import CalledProcessError
-from typing import Dict, List, Union, Optional
+from typing import Dict, List, Union, Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from snapcraft.internal.pluginhandler._dirty_report import DirtyReport
+    from snapcraft.internal.pluginhandler._outdated_report import OutdatedReport
+
+
+"""Commonly used resolution message to clean and retry build."""
```

Comment:
this should be a code comment (#), as it is now, it is seen as the module's docstring

Reply:
I was thinking it should be doc'd like that as it could be useful for other error modules too... but I'm indifferent so I changed to a code comment anyhow :)

Reply:
Ah, but this would be the help for the module, not the string following.